crowbar_framework/app/models/node_object.rb

```diff
@@ -134,15 +134,21 @@ def self.find_node_by_public_name(name)
 
   def self.find_node_by_name(name)
     name += ".#{ChefObject.cloud_domain}" unless name =~ /(.*)\.(.)/
-    val = begin
-      Chef::Node.load(name)
+    begin
+      chef_node = Chef::Node.load(name)
+      unless chef_node.nil?
+        NodeObject.new(chef_node)
+      else
+        nil
+      end
     rescue Errno::ECONNREFUSED => e
       raise Crowbar::Error::ChefOffline.new
     rescue StandardError => e
       Rails.logger.warn("Could not recover Chef Crowbar Node on load #{name}: #{e.inspect}")
       nil
+    rescue Crowbar::Error::NotFound => e
```

Comment:
@bkutil: hrm, I don't think we want the warning in that case; I guess just moving the rescue a few lines above should help with this?